signUp.js

```diff
@@ -0,0 +1,33 @@
+import { PrismaClient } from '@prisma/client';
+const prisma = new PrismaClient()
+
+
+const signUp = async(req, res) => {
+  try {
+    
+    const { email, name, password } = req.body
+
+    const users = await prisma.$queryRaw(`
+      SELECT * FROM users WHERE email = '${email}'
+    `)
+    
+    if (users) {
+      res.status(400).json({ message: 'ALREADY_EXCISTING_USER'})
+    }
+
+    const CreateUser = await prisma.$queryRaw(`
+      INSERT INTO users (email, name, password) VALUES ('${email}', '${name}', '${password}');

```

Comment:
```suggestion

    const createdUser = await prisma.$queryRaw(`
      INSERT INTO users (email, name, password) VALUES ('${email}', '${name}', '${password}');

```
클래스가 아닌 변수는 대문자로 시작하지 않습니다 :)